Dockerfile

```diff
@@ -1,5 +1,6 @@
 # syntax=docker/dockerfile:experimental
-FROM ubuntu:20.04 as baseline
+FROM ubuntu:22.04 as baseline
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
Need to validate these items. If needed we'll have to update all Ubuntu image baselines from 20.04 LTS -> 22.04 LTS.

Reply:
I will be doing docker based testing, the updated image is needed for skopeo.